partiql-planner/src/main/kotlin/org/partiql/planner/internal/transforms/RexConverter.kt

```diff
@@ -851,7 +851,12 @@ internal object RexConverter {
                 DataType.CHARACTER, DataType.CHAR -> call(FunctionUtils.OP_IS_CHAR, targetType.length.toRex(), arg0)
                 DataType.CHARACTER_VARYING, DataType.VARCHAR -> call(FunctionUtils.OP_IS_VARCHAR, targetType.length.toRex(), arg0)
                 DataType.CLOB -> call(FunctionUtils.OP_IS_CLOB, arg0)
-                DataType.STRING -> call(FunctionUtils.OP_IS_STRING, targetType.length.toRex(), arg0)
+                DataType.STRING -> {
```

Comment:
self-review: the function calls for `IS STRING` and `IS DECIMAL`/`IS NUMERIC` without any type arguments parameters were getting mistranslated to the wrong function call

Reply:
previously: IS STRING -> IS STRING(<null>)
fixed: IS STIRNG -> IS STRING

similarly for decimal
previously: IS DECIMAL -> IS DECIMAL(<null>, <null>)
fixed: IS DECIMAL -> IS DECIMAL